### DISCUSSION TITLE
Mask Image not recognised

### DISCUSSION THREAD

**Discussion by cddmastro** [aaronwmorris/indi-allsky#1436] [General]
Hi, Loving Indi-allsky, trying to use a mask as I have a lot of trees around my camera. When trying to add to config I get a red image all black message. Any ideas what I have wrong ?. Many Thanks
![ABMask (1)](https://github.com/user-attachments/assets/d0fc8eb7-0544-4011-88ee-fecb33881c31)
![allsky-startrail_ccd1_20240728_night](https://github.com/user-attachments/assets/b170fe86-3163-43df-b7f8-ce565e6443c4)

**Comment by aaronwmorris** [comment]
That looks correct.  Do you have the mask configured as the "detection mask" or the "logo overlay"?  It should be the logo overlay if you are trying to mask out the trees.

**Reply by cddmastro** [reply]
Thanks for the response, not sure what the difference. will try logo overlay and see what I get. Thanks again

**Reply by aaronwmorris** [reply]
The detection mask only masks areas for things like ADU (brightness), star, and meteor calculations, but it does not alter the final image.  The detection mask only sets the boundary for the detection area.

**Reply by morganjrees** [reply]
Where in settings do I find detection mask to exclude my trees and buildings

**Reply by aaronwmorris** [reply]
@morganjrees The Detection Mask setting is on the Processing tab near the top.

**Reply by morganjrees** [reply]
@aaronwmorris do I download the mask image and create a png in something like paint and upload it here?

**Reply by aaronwmorris** [reply]
You can use the mask base image as the starting point.  Once you are finished, you would upload the mask to the indi-allsky server through SFTP and then you specify the path to the mask in the config field.
